linera-views/src/common.rs

```diff
@@ -174,6 +202,277 @@ pub trait KeyValueStoreClient {
     }
 }
 
+/// A key-value store that splits the big values into multiple keys.
+/// Some databases have a design with a relatively small size for values (like 400 kB)
```

Comment:
nit: insert newline after the first sentence and before the rest of the explanations.

Reply:
Please avoid the vague language "some databases", "relatively small". Let's be concrete.

Reply:
I thought that we did not want to say too precise statement. So, I added a sentence about `DynamoDb` mention.